src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/Home';
+import './App.css';
+import Layout from './components/Layout';
+import Signup from './components/Signup';
+import Login from './components/Login';
+import Details from './components/Details';
+import Reserve from './routes/Reserve';
+import MyReservations from './components/MyReservations';
+import AddMotorbikeForm from './components/AddMotorbikeForm';
+import DeleteMotorbike from './components/DeleteMotorbike';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor these functions to ES6 arrow functions, so your code complies with the latest standards.


![image](https://github.com/SaneMyburg/motorbike-front-end/assets/57421823/b1001b83-d285-42e4-83c5-015713a5dac8)


Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪